### ISSUE TITLE
oj.IntlConverterUtils.dateToLocalIso not conforming to date ISO8601?

### ISSUE THREAD

**Issue by janis-kra** [oracle/oraclejet#9]
Hey there,

we are writing a webapp in which we have a CRUD dialog that allows the user to modify some data that comes from a database webservice. Amongst others, the user can view and edit a date value via an ojInputDateTime input component. To use this component, we have to convert the date strings that the webservice returns by calling `oj.IntlConverterUtils.dateToLocalIso`.

So far, so good, but the problem is that after converting the date strings, the time is off by two hours. So for example, when executing the following line of code, the time is 23:33:07 instead of 21:33:07

``` js
oj.IntlConverterUtils.dateToLocalIso(new Date('2016-06-21T21:33:07.111Z'))
```

According to the ISO standard for dates (https://en.wikipedia.org/wiki/ISO_8601#Times) the trailing Z stands for UTC time (this is how our database represents date strings). Thus, the hour should remain 21 and not 23 if I'm not mistaken.

Long story short, this can be fixed if the trailing Z is removed manually before passing the string into `dateToLocalIso`. Nevertheless, I think this is a bug and should be fixed sometime in the future.

(by the way, I deliberately posted this here and **not** in the OTN forums, because I think this repository could use some more activity - sorry for the inconvenience ;) )

**Comment by peppertech** [comment]
Hi Janis,

Thanks for posting.  This is indeed a case of where Oracle JET does not support timezones at the moment.  We already have an ER in the works to provide this support, and hope to have it available as soon as possible.

You have correctly stated the workaround for this right now.  Manually strip the Z before passing the value in.

**Comment by peppertech** [comment]
Hi Janis,

A follow up to the original issue. I talked to engineering and this is what they replied with....

This is not a bug. new Date('2016-06-21T21:33:07.111Z') will return a date in the local system timezone. It appears you are 2 hours ahead of UTC, so new Date('2016-06-21T21:33:07.111Z') returns 23 as hours and we construct from that into an ISO string.

The rest of the comment about adding the timezone support goes toward the new support we will be adding in our converter and format features in the toolkit.

Hope that makes sense.

Thanks again for taking the time to write it up and send it to us.

**Comment by janis-kra** [comment]
I'm not quite sure if I would want the converter to return a date in my local timezone if I explicitly give it a date in UTC, but as that is the behavior intended by the programmers, it kind of makes sense. Thanks for the follow up!
